AndelaOSP/Andela-Socials
# #165853335 Redesign event card

#### What Does This PR Do?
   This PR redesigns the event card on the dashboard to have a modern look and feel

#### Description Of Task To Be Completed
  [x]- Make slack icon component reusable
  [x]- Add image section
  [x]- Add details section
  [x]- Add slack share button to cards

#### Any Background Context You Want To Provide?
None

#### How can this be manually tested?
Switch to branch `ft-redesign-event-card-165853335` and login, the first page should contain the cards

#### What are the relevant pivotal tracker stories?
[#165853335](https://www.pivotaltracker.com/story/show/165853335)

#### Screenshot(s)
<img width="1679" alt="Screenshot 2019-05-12 at 4 23 51 AM" src="https://user-images.githubusercontent.com/19429611/57577496-1c367180-7470-11e9-9374-0ed086922297.png">
